### PR TITLE
remove CMS-signed-CBOR from document: no longer needed

### PR DESCRIPTION
**Pull request by mcr**

### CODE REVIEW

File: constrained-voucher.mkd [hunk 1]
```diff
@@ -118,25 +118,20 @@ Rather than serializing the YANG definition in JSON, it is serialized into CBOR
 
 This document follows a similar, but not identical structure as {{RFC8366}} and supplements the brski part to {{I-D.ietf-ace-coap-est}}.
 
-There are three constrained situations described in this document:
-1. CMS signed CBOR encoded vouchers transported using CoAP, protected by DTLS (coaps).
-2. COSE signed CBOR encoded vouchers transported using CoAP, protected by EDHOC or DTLS.
-3. COSE signed CBOR encoded vouchers, integrated into the key exchange as described by {{!I-D.selander-ace-ake-authz}}
+This document describes a single new voucher format.
+It is a COSE-signed CBOR-encoded vouchers transported using CoAP.
```
Comment (EskoDijk) [inline constrained-voucher.mkd:122]: vouchers -> voucher
the transport can also be HTTPS, from MASA to Registrar; not only CoAP.

File: constrained-voucher.mkd [hunk 2]
```diff
@@ -118,25 +118,20 @@ Rather than serializing the YANG definition in JSON, it is serialized into CBOR
...
+The CoAP connection is protected either by OSCORE+EDHOC, or DTLS (CoAPS).
+
+There is additional work where the voucher is integrated into the key-exchange, described in {{!I-D.selander-ace-ake-authz}}.
```
Comment (EskoDijk) [inline constrained-voucher.mkd:125]: work where -> work when

File: constrained-voucher.mkd [hunk 3]
```diff
@@ -705,182 +655,6 @@ A 2.04 Changed response returning CBOR voucher signed with a cms structure(ct=TB
 ~~~~
 INSERT_FIG_FROM_FILE examples/cms-voucher2.b64 END
```
Comment (EskoDijk) [inline constrained-voucher.mkd:656]: above INSERT lines include CMS request/voucher, so: to be removed!

**Review by EskoDijk — APPROVED** [review]
I've provided a few comments in the PR, for further updates. Overall the proposed change looks ok. (In case we missed some CMS related items, these can always be removed later during future reviews.)

File: constrained-voucher.mkd (hunk 1 above)
Comment (mcr) [inline constrained-voucher.mkd:122]:
```suggestion
It is a COSE-signed CBOR-encoded voucher transported using secured CoAP or HTTP.
```

File: constrained-voucher.mkd (hunk 2 above)
Comment (mcr) [inline constrained-voucher.mkd:125]:
```suggestion
There is additional work when the voucher is integrated into the key-exchange, described in {{!I-D.selander-ace-ake-authz}}.
```

File: constrained-voucher.mkd (hunk 3 above)
Comment (mcr) [inline constrained-voucher.mkd:656]:
```suggestion
~~~~
INSERT_FIG_FROM_FILE examples/cms-voucher2.b64 END
```

**Pull request merged**